tests/testthat/test_GLA_pipeline.R

```diff
@@ -0,0 +1,238 @@
+context("Run data quality check on Glasgow, Scotland, pipeline output")
```

Comment:
All passing for me now. Once this is ready we'll merge in the master and adjust the setup so it works with the new testing architecture.